src/mmw/sass/components/_modals.scss

```diff
@@ -138,3 +138,20 @@
   top: 1rem;
   left: 2rem;
 }
+
+.fa {
+  font-size: 1.0rem;
+  padding: 0 0.1rem 0 0.1rem;
+}
```

Comment:
I might have the wrong idea about how the sass rules are scoped. Do those rules apply to components other than modal elements?